apps/dashboard/app/views/layouts/_footer.html.erb

```diff
@@ -1,20 +1,15 @@
 <footer>
-  <div class="container-fluid">
-    <div class="row">
-      <div class="col-md-6 col-sm-6">
-        <%= link_to "https://openondemand.org" do %>
-          <%= image_tag(
-                "OpenOnDemand_powered_by_RGB.svg",
-                class: "footer-logo",
-                alt: "Powered by Open OnDemand",
-                height: "40",
-                style: "margin-bottom: 20px"
-              ) %>
-        <% end %>
-      </div>
-      <div class="col-md-6 col-sm-6">
-        <p class="pull-right">OnDemand version: <%= Configuration.ood_version %></p>
-      </div>
-    </div>
-  </div><!-- /.container -->
-</footer>
+  <div class="d-flex justify-content-between align-items-center px-3 py-3">
+    <%= link_to "https://osc.github.io/Open-OnDemand/" do %>
```

Comment:
Looks like this link needs updated.

Reply:
Resolved in https://github.com/OSC/ondemand/pull/927/commits/d5335e322ceb11a36f133682534469f7d476f255